### PR TITLE
Provide options#81

### PR DESCRIPTION
**Pull request by sculove**
## Issue
#81 

## Details
- add hammerMangerOptions on Pan/PinchInput
- removes inputType from Axes for tree shaking
- update dependency

### CODE REVIEW

**Comment by jongmoon** [comment]
I'll check it after afternoon

File: src/inputType/PanInput.ts [hunk 1]
```diff
@@ -149,9 +161,12 @@ export class PanInput implements IInputType {
...
 			}
-			this.hammer = createHammer(this.element,
-				[Hammer.Pan, hammerOption],
-				inputClass);
+			this.hammer = createHammer(this.element, { ...{
+				recognizers: [
+					[Hammer.Pan, hammerOption],
+				],
+				inputClass,
+			}, ... this.options.hammerManagerOptions });
```
Comment (jongmoon) [inline src/inputType/PanInput.ts]: I think `hammerManagerOptions` is somewhat ambiguous. 

In above comment, it looks `hammerManagerOptions` can have`{cssProps: {userSelect: "none",touchSelect: "none",touchCallout: "none",userDrag: "none"}. 

But in this code, you can replace recognizers, inputClass with new one `this.options.hammerManagerOptions`
```js
}, ... this.options.hammerManagerOptions });
```

File: src/index.ts [hunk 2]
```diff
@@ -1,3 +1,12 @@
 import Axes from "./Axes";
+import { PanInput } from "./inputType/PanInput";
+import { PinchInput } from "./inputType/PinchInput";
+import { WheelInput } from "./inputType/WheelInput";
+import { MoveKeyInput } from "./inputType/MoveKeyInput";
+
+Axes.PanInput = PanInput;
+Axes.PinchInput = PinchInput;
+Axes.WheelInput = WheelInput;
+Axes.MoveKeyInput = MoveKeyInput;
```
Comment (jongmoon) [inline src/index.ts]: Why seperate them from Axes?

File: src/inputType/PanInput.ts (hunk 1 above)
Comment (sculove) [reply src/inputType/PanInput.ts]: @jongmoon you're right. the hammerManager is that for advance users.
the structure of hammerManager options is so complex. if user want to change inputType or recognize, user can chage it.

File: src/index.ts (hunk 2 above)
Comment (sculove) [reply src/index.ts]: @jongmoon for tree shaking!!!

**Pull request merged**